docs/tutorials/projects/bedtime-songs-bot.md

```diff
@@ -5,7 +5,7 @@ weight: 60
 type: "docs"
 tags: ["mac", "app", "board", "webcam", "camera", "ml", "machine learning", "babysitter"]
 description: "Create a robot babysitter with a webcam and machine learning."
-imageAlt: "Image of Tess and their computer."
+images: [ "/tutorials/img/bedtime-songs-bot/robot_babysitter.gif" ]
```

Comment:
you need to move the gif into the folder `static/tutorials/img/bedtime-songs-bot` then if you change it to this